bindings/wasm/src/tangle/config.rs

```diff
@@ -1,204 +0,0 @@
-// Copyright 2020-2022 IOTA Stiftung
```

Comment:
Do we want to keep this empty file?

Reply:
Thanks for the catch, I think a merge re-added it.